### PR TITLE
Ammonite should not show up if it's on bintray

### PR DESCRIPTION
**Pull request by alexdeleon**
This resolves the first part of issue #256. Can be tested with project `marpiec/mpjsons` which is on Bintray only

### CODE REVIEW

File: model/src/main/scala/ch.epfl.scala.index.model/Release.scala [hunk 1]
```diff
@@ -201,6 +202,14 @@ case class Release(
 
     orderedReverseDependencies.filter(d => d.reference.name == dep.reference.name).map(_.reference.version)
   }
+
+  /**
+    * True is these release artifact is hosted on Bintray but not Maven central
+    */
+  lazy val isHostedOnBintray = resolver.map {
```
Comment (MasseGuillaume) [inline model/src/main/scala/ch.epfl.scala.index.model/Release.scala]: it could be `def isOnMavenCentral = resolver.isEmpty`

Comment (alexdeleon) [inline model/src/main/scala/ch.epfl.scala.index.model/Release.scala]: Agree, it's better like that

**Comment by MasseGuillaume** [comment]
@alexdeleon hehe really sorry about that. Thanks for staying in the loop. If you need help I'm always in the gitter.im channel: https://gitter.im/scalacenter/scaladex

**Comment by MasseGuillaume** [comment]
Hey @alexdeleon, I really want to merge this, do you need help?

**Comment by MasseGuillaume** [comment]
I'm closing this PR. We can reopen it later ;-).

**Pull request closed without merging**